### PR TITLE
VER: Release 0.5.0

### PR DESCRIPTION
**Pull request by cjdsellers**
See release notes.

### CODE REVIEW

File: databento/historical/client.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Historical:
     key : str, optional
         The user API key for authentication.
         If ``None`` then the `DATABENTO_API_KEY` environment variable is used.
-    gateway : HistoricalGateway or str, default HistoricalGateway.NEAREST
+    gateway : HistoricalGateway or str, default HistoricalGateway.BO1
```
Comment (zcqian) [inline databento/historical/client.py:23]: is this the default we settled on?

File: databento/historical/error.py [hunk 2]
```diff
@@ -13,18 +16,18 @@ class BentoHttpError(BentoError):
...
         super(BentoHttpError, self).__init__(message)
 
-        if http_body and hasattr(http_body, "decode"):
+        if http_body and isinstance(http_body, bytes):
             try:
                 http_body = http_body.decode("utf-8")
-            except BaseException:
+            except UnicodeDecodeError:
                 http_body = (
                     "<Could not decode body as utf-8. "
```
Comment (zcqian) [inline databento/historical/error.py:32]: would be nicer to give more context -- easier for us as well

File: databento/historical/client.py (hunk 1 above)
Comment (cjdsellers) [reply databento/historical/client.py:23]: Yes, for now. As the concept of nearest for historical doesn't make sense with where we'll be hosting the data.

File: databento/historical/error.py (hunk 2 above)
Comment (zcqian) [inline databento/historical/error.py:32]: would be nicer to give more context

Comment (cjdsellers) [reply databento/historical/error.py:32]: Agreed

File: databento/historical/http.py [hunk 3]
```diff
@@ -185,7 +179,7 @@ def _stream(
     async def _stream_async(
         self,
         url: str,
-        params: List[Tuple[str, Optional[str]]],
+        params: Sequence[Tuple[str, Optional[str]]],
```
Comment (zcqian) [inline databento/historical/http.py:182]: Is there a particular reason that we want `List` in other places but a `Sequence` will suffice here? (I know Python type hints can be annoying sometimes)

**Review by zcqian — APPROVED** [review]
I do have a few questions, though.

**Pull request merged**